Documentation/Development.md

```diff
@@ -77,6 +77,8 @@ using a [snapshot](https://swift.org/download/#releases) from swift.org.
 	$ Utilities/bootstrap test --test-parallel
 	```
 	Use this command to run the tests. All tests must pass before a patch can be accepted.
+
+	If you encounter issues building or testing the Swift Package Manager project, try installing the latest version of the Xcode beta if applicable.
```

Comment:
Its always a good idea to have latest released Xcode for developing Swift OSS projects. So, instead of adding this line here, how about we move the line `First, follow the instructions provided here` from `Using Swift compiler build script` section to its own section called "Initial setup". That link takes you to system requirements page of Swift compiler, which lists Xcode version and other system dependencies https://github.com/apple/swift/blob/master/README.md#system-requirements

Reply:
I'm worried that people will follow that link and try to do other stuff on that page, like install CMake and Ninja, which is not necessary for working on SwiftPM. Wouldn't the least confusing solution just to reiterate here that the latest Xcode is necessary?

Reply:
Having CMake and Ninja is good IMO. Its harmless and you might need to build llbuild with SwiftPM sometimes

Reply:
What if people get confused that you need to build the compiler? I had people confused at try Swift about that, even without a link to the compiler documentation.

Reply:
In that case, I suggest we improve this documentation and clearly state what needs to be done. I don't want to repeat the information because Swift readme might add or remove dependencies and we might get out-of-date.